src/community/script/core/src/test/java/org/geoserver/script/wps/ScriptProcessIntTest.java

```diff
@@ -35,6 +35,7 @@ public void testMapResult() throws Exception {
                  "</wps:Execute>";
           
          Document doc = postAsDOM("wps", xml);
+         print(doc);
```

Comment:
@jodygarnett @aaime do we still discourage use of GeoServerSystemTestSupport.print(Document) (etc) since -DquietTests was introduced? I think so. @NielsCharlier we like tests to not flood the console, so perhaps this line should be removed.